frontend/src/components/routes.tsx

```diff
@@ -118,25 +119,29 @@ function EmitRouteViews(entries: IRouteEntry[]): JSX.Element[] {
 
 export const RouteView = (() =>
     <AnimatePresence exitBeforeEnter>
-        <Switch>
+        <Routes>
             {/* Index */}
-            {/* <Route exact path='/' component={IndexPage} /> */}
-            <Route exact path="/" render={() => <Redirect to="/topics" />} />
+            {/* <Route path='/' component={IndexPage} /> */}
+            <Route path="/" element={<Navigate to="/topics" />} />
 
             {/* Emit all <Route/> elements */}
             {EmitRouteViews(APP_ROUTES)}
 
-            <Route render={rp => {
+            {/* <Route element={rp => {
                 uiState.pageTitle = '404';
                 return (
-                    <Section title="404">
-                        <div><h4>Path:</h4> <span>{rp.location.pathname}</span></div>
-                        <div><h4>Query:</h4> <pre>{JSON.stringify(rp.location.search, null, 4)}</pre></div>
-                    </Section>
-                )
-            }} />
 
-        </Switch>
+                )
+            }} /> */}
+            <Route element={
+                <Section title="404">
+                    <>{uiState.pageTitle = '404'}</>
```

Comment:
Nasty workaround since the route is no longer accepting a callback, just a component directly